src/mmw/mmw/settings/base.py

```diff
@@ -57,7 +57,9 @@ def get_env_setting(setting):
 
 
 # FILE STORAGE CONFIGURATION
-DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+DEFAULT_FILE_STORAGE = environ.get('MMW_DEFAULT_FILE_STORAGE', 'storages.backends.s3boto3.S3Boto3Storage')
```

Comment:
If the expectation is that this storage backend will be used across all environments, does it make sense to bypass the environment variable configuration parameter and just leave `DEFAULT_FILE_STORAGE` assigned to `storages.backends.s3boto3.S3Boto3Storage`?

Reply:
That makes sense. After the discussion @rajadain's thread, I decided to make development use the default file storage backend, and leave `DEFAULT_FILE_STORAGE` assigned to `storages.backends.s3boto3.S3Boto3Storage` in the production settings module.